### ISSUE TITLE
Lightbox steals keyboard events from PLYR.

### ISSUE THREAD

**Issue by Lefthandmedia** [biati-digital/glightbox#169]
**Describe the bug**
The PLYR is not accesible for keyboard users. Plyr it self has great a11y keyboard shortcuts. They are just not usable  because Glightbox 'steals' the keyboard events.
This could be a feature too

**Are you able to reproduce the bug in the demo site**
Yes

**To Reproduce**
- place an video in GL
- try to stop the video with the spacebar (or K key).... no luck

**Expected behavior**
Video should stop

**Desktop:**
 - Browser all

**Comment by Lefthandmedia** [comment]
Fixed this by adding
```  
lightbox.on('slide_changed', ({ current }) => {
    const { player } = current;
    if (player) {
      player.elements.container.focus();
      }
  });
```

**Comment by biati-digital** [comment]
As you mention in your fix, the keyboard it's not working simply because it has no focus, if you focus the player all the player keyboard shortcuts will work correctly.  I'm not planning to change the default functionality as i'm sure there will be people creating issues about why the left and right arrow does not change slides. 

For now i've create a new option called "autofocusVideos" to give automatic focus when the video starts playing so the keyboard shortcuts work correctly for the video player.

Please download the latest version and try it.

**Comment by Adstokoe19** [comment]
@biati-digital The new option "autofocusVideos" doesn't work for me: https://codepen.io/adstokoe/pen/BarmoOq

Also, if this was working, why would you deactivate the prev and next arrows to change slides? 

The lightbox in it's current state is failing on accessibility standards if used for video: 
https://www.w3.org/WAI/media/av/player/#player-accessibility-functionality 
https://www.w3.org/WAI/WCAG21/quickref/#keyboard-accessible (Level A failing)

The ply.io library is accessible but not when it's used with Glightbox.

**Comment by biati-digital** [comment]
I've no idea what you mean by `why would you deactivate the prev and next arrows to change slides?` i did not deactivate the prev and next arrows, did you check the code? the option "autofocusVideos" the only thing it does is try to focus the video and that's it.  This is the actual code.

```javascript
if (player && !player.playing) {
    player.play();
    if (this.settings.autofocusVideos) {
        player.elements.container.focus();
    }
}
```

You can try to use the code posted by Lefthandmedia or you can improve the integration with the player and provide a PR.
